modules/projectLimeLightBidAdapter.js

```diff
@@ -0,0 +1,125 @@
+import { registerBidder } from 'src/adapters/bidderFactory';
+import { BANNER, VIDEO } from 'src/mediaTypes';
+import {ajax} from '../src/ajax';
+import * as utils from 'src/utils';
```

Comment:
In lieu of this PR #3435, any imported files within a modules file needs to use the relative style path.

Can you please update these imports accordingly?

Reply:
Fixed. Thanks for review